types/@metamask/eth-hd-keyring.d.ts

```diff
@@ -0,0 +1 @@
+declare module '@metamask/eth-hd-keyring';
```

Comment:
Note that adding these files will make it possible to build `@metamask/keyring-controller`. But these type augmentations won't get included in the compiled version of the package, so in order to use the package, consumers may have to add these to their project as well.

However... I see that we copied these from `eth-keyring-controller`. And obviously we haven't had the problem I described above, because otherwise we would have had to handle it already. So maybe  we don't need to do anything right now, but we should probably make a note to add type definition files to these packages (even if they're empty) so we don't have to do this.

Reply:
Uhm good point. We should tackle the typescript migration for these keyring packages, and that might be part of the effort to align them all with the Keyring API - but in the meantime, adding blank type definition files to these packages sounds like a good short-term solution.